app/views/scores/index.html.erb

```diff
@@ -1,7 +1,11 @@
-<h1><%= @score.title %></h1>
-
+<h1 id='score-title'><%= @score.title %></h1>
+<select id='scores-menu' onchange="window.location=this.value">
+  <option>Select a Score</option>
+  <% @score_bank.all_scores.each do |score| %> 
+    <option value="/scores?score_id=<%= score.id %>"><%= score.title %></option>
+  <% end %>
+</select>
```

Comment:
I wonder if we could find a way to reduce the loading time more, this definitely provides ease to navigation

Reply:
Also this may become hard to navigate if we have a lot of scores on one user